moulinette/interfaces/cli.py

```diff
@@ -335,6 +335,7 @@ def run(self, args, output_as=None, password=None):
                 - plain: return a script-readable output
                 - none: do not output the result
             - password -- The password to use in case of authentication
+            - timeout -- Timeout this command after 30 seconds if the lock isn't acquired
```

Comment:
also, reading the code, 0 does not mean "no timeout" as one could expect, so might be worth mentioning here.